Source/WebKit/UIProcess/mac/WKImmediateActionController.h

```diff
@@ -56,9 +56,9 @@ enum class ImmediateActionState {
 
 @interface WKImmediateActionController : NSObject <NSImmediateActionGestureRecognizerDelegate> {
 @private
-    NakedPtr<WebKit::WebPageProxy> _page;
+    CheckedPtr<WebKit::WebPageProxy> _page;
```

Comment:
In general, with ObjC objects, it might be a bit risky to assume that their lifetimes are always limited to that of our C++ objects due to `-autorelease`, and the fact that any other framework code can add a `-retain` / `-release` from underneath us.

However, I think it's correct in this case since `WebViewImpl` maintains full ownership over `WKImmediateActionController` — the only other platform object that references this, `NSImmediateActionGestureRecognizer`, only holds a weak delegate pointer).

Reply:
Yeah, it's a little subtle here but should be okay.